src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.resx

```diff
@@ -292,7 +292,7 @@
     <value>7</value>
   </data>
   <data name="localsCommandButton.Text" xml:space="preserve">
-    <value>&amp;Clear All NuGet Cache(s)</value>
+    <value>&amp;Clear All NuGet Storage</value>
```

Comment:
There are many options, other than "storage". My personal preference was storage is the best.

Other options I thought about was "directories" or "folders".  If anyone has a different preference or idea, please share.

Reply:
Would it be too verbose to say “Remove all packages and HTTP cache”?  I feel something more precise and unambiguous would be best. But I’m having picturing what the button would look like with so many words 😅

Reply:
Technically it will also clear the plugins cache and "temp" (NuGetScratch) folder, not just the packages and http cache folders.  But these other two typically use very little disk space in comparison.